force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -139,15 +197,12 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static String returnHelloWorld() {
         // Initialize the result String
-
-        // Use a for loop to append 'Hello World!;' to the result string three times
-        /* 
-         for (){
-            
-         }
-        */
-
-        return null; // Replace null with the variable you used to store the result
+        String a = 'Hello World!; ';
+        String result = '';
+        for (Integer b = 0; b < 3; b++) {
```

Comment:
Nicely done. It is best practice to use "i" to represent the Integer in a traditional for loop